synthesizer/process/src/cost.rs

```diff
@@ -60,7 +60,14 @@ pub fn deployment_cost<N: Network>(deployment: &Deployment<N>) -> Result<(u64, (
 /// Returns the *minimum* cost in microcredits to publish the given execution (total cost, (storage cost, finalize cost)).
 pub fn execution_cost<N: Network>(process: &Process<N>, execution: &Execution<N>) -> Result<(u64, (u64, u64))> {
     // Compute the storage cost in microcredits.
-    let storage_cost = execution.size_in_bytes()?;
+    let mut storage_cost = execution.size_in_bytes()?;
+
+    // Compute a quadratic storage cost penalty if above the size penalty threshold.
+    if storage_cost > N::EXECUTION_STORAGE_PENALTY_THRESHOLD {
```

Comment:
I would strongly advise introducing unit tests that explicitly **clamp** the expected upper bound and lower bound numeric value behaviors prior to the approval and merge of this PR.

Reply:
These should be able to added before the end of this week.

Reply:
@iamalwaysuncomfortable what's the status of these unit tests? Per @howardwu recommendation, we don't want to merge until we get those tests included.

Reply:
@apruden2008 They'll be committed today.